snuba/settings/__init__.py

```diff
@@ -155,7 +155,7 @@
 COLUMN_SPLIT_MAX_RESULTS = 5000
 
 # Migrations in skipped groups will not be run
-SKIPPED_MIGRATION_GROUPS: Set[str] = {"querylog", "spans_experimental", "profiles"}
+SKIPPED_MIGRATION_GROUPS: Set[str] = {"querylog", "spans_experimental"}
```

Comment:
Are you sure we are ready to remove from this list already? Remember this will go out to all self hosted users the moment we do this.

Reply:
I guess I'm confused. Why do we have `OPTIONAL_GROUPS` and `SKIPPED_MIGRATION_GROUPS`? I want to deploy that in snuba production but not anywhere else.

Reply:
Sorry it is a bit confusing. `OPTIONAL_GROUPS` is hardcoded in Snuba's codebase and cannot be changed. Anything in this file (incl. SKIPPED_MIGRATION_GROUPS) can be overridden by the user when they are running snuba. So the groups that are allowed to be optional are hardcoded, but the ones that are actually skipped can be reconfigured by the user. I think you should still skip `profiles` by default here. But you can actually override this value in `ops` if you want to roll it out to Snuba production.

Reply:
Alright, sounds good.

Reply:
Are you planning to put profiles back here? I think it should stay in the default settings until it is ready for roll out in every environment.

Reply:
Oh right, I need that skipped.